server/app.js

```diff
@@ -31,14 +32,15 @@ app.use(cors());
 app.use('/', indexRouter);
 app.use('/ping', pingRouter);
 app.use('/api/user', user);
+app.use('/api/fund', fund);
 
 // catch 404 and forward to error handler
-app.use(function(req, res, next) {
+app.use(function (req, res, next) {
```

Comment:
looks great, looks like you're using a linter now :)

Reply:
Yups. ;)